ccmlib/scylla_node.py

```diff
@@ -1240,6 +1240,12 @@ def flush(self):
         self.nodetool("flush")
         self._wait_no_pending_flushes()
 
+    def get_release_major_scylla_version(self):
```

Comment:
Would be useful to have a comment on this, saying that this returns something like "4.5".
Is "major version" the official name we use for something like "4.5"?

A more serious issue: how does this work for enterprise versions?